LanguageFeatures/Extension-types/static_analysis_extension_types_A02_t01.dart

```diff
@@ -22,19 +22,19 @@ extension type ET1(int id) {
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  ET1.n2(int id) : this();
+  ET1.n2(int id) : this(id);
 //^^^
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  ET1.n3(int id) : this.new();
+  ET1.n3(int id) : this.new(id);
```

Comment:
Similarly here. This line doesn't look like a compile-time error to me.

Reply:
Agreed again.